### PR TITLE
add smtp setting example: 126 email used widely in China

### PR DESCRIPTION
**Pull request by lfckop**
add smtp setting example: 126 email used widely in China

### CODE REVIEW

File: doc/settings/smtp.md [hunk 1]
```diff
@@ -76,6 +76,21 @@ gitlab_rails['smtp_authentication'] = "login"
...
+gitlab_rails['smtp_enable'] = true
+gitlab_rails['smtp_address'] = "smtp.126.com"
+gitlab_rails['smtp_port'] = 25
+gitlab_rails['smtp_user_name'] = "your_email@126.com"
+gitlab_rails['smtp_password'] = "your_smtp_authorization_code"
+gitlab_rails['smtp_domain'] = "126.com"
+gitlab_rails['smtp_authentication'] = :login
+gitlab_rails['smtp_enable_starttls_auto'] = true
+gitlab_rails['gitlab_email_from'] = 'your_email@126.com'
+user["git_user_email"] = "your_email@126.com"
```
Comment (maxlazio) [inline doc/settings/smtp.md]: @lfckop Can you tell me why is this necessary?

**Comment by maxlazio** [comment]
@lfckop Thanks for your contribution! Can you check my line comment?

**Comment by lfckop** [comment]
@maxlazio Just add an example, 126 email is widely used in China, and this example may save others' time. For some reasons, we can't access gmail in China. Thank you.

**Comment by Razer6** [comment]
I think @maxlazio was refering to the line `user["git_user_email"] = "your_email@126.com"`

**Comment by maxlazio** [comment]
@Razer6 Thanks for making it clearer, yes that is what I meant. @lfckop can you remove the line with `user["git_user_email"] = "your_email@126.com"`?

**Comment by lfckop** [comment]
@maxlazio @Razer6  From our installation experience, gitlab_rails['smtp_user_name']  , gitlab_rails['gitlab_email_from']   and user["git_user_email"]  should keep consistent, or it'll fail in replying email like register and comment remind. So, I don't thing it's appropriate to remove the line. Thanks!
